core/src/main/scala/org/http4s/internal/package.scala

```diff
@@ -355,4 +358,15 @@ package object internal {
       tail: Eval[Int]*
   ): Int =
     reduceComparisons_(NonEmptyChain(Eval.now(head), tail: _*))
+
+  def pairsToMultiParams[K, V](map: collection.Seq[(K, Option[V])]): IMap[K, ISeq[V]] =
```

Comment:
Getting this out of the condemned traits.  I guess it should go right on `Query` instead.